### PR TITLE
Add typings for is-alphanumerical

### PR DESCRIPTION
**Pull request by vutran**
This PR will add a definition for the npm package [`is-alphanumerical`](https://www.npmjs.com/package/is-alphanumerical).

The author prefers the typings to be sourced in DT rather than the repo itself ([ref](https://github.com/wooorm/is-alphanumerical/pull/7)).

### CODE REVIEW

File: is-alphanumerical/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+// Type definitions for is-alphanumerical 1.0
+// Project: https://github.com/wooorm/is-alphanumerical/
+// Definitions by: Vu Tran <https://github.com/vutran>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare function isAlphanumerical(val: string | boolean | string): boolean;
```
Comment (paulvanbrenk) [inline is-alphanumerical/index.d.ts]: Think you can drop one of the `strings` here... did you mean `number`

**Comment by paulvanbrenk** [comment]
Looks good, other than the question I have.

**Comment by vutran** [comment]
Ah, sorry for the typo. Should be fixed now.

**Comment by Dru89** [comment]
Was this meant to go in the `types` folder?

**Comment by paulvanbrenk** [comment]
Good catch

**Pull request merged**